qa/zcash/postponed-updates.txt

```diff
@@ -5,21 +5,23 @@
 #
 
 # Ccache 4.0 requires adding CMake to the depends system.
-native_ccache 4.0 2021-10-01
-native_ccache 4.1 2021-10-01
-native_ccache 4.2 2021-10-01
-native_ccache 4.2.1 2021-10-01
-native_ccache 4.3 2021-10-01
-native_ccache 4.4 2021-10-01
-native_ccache 4.4.1 2021-10-01
-native_ccache 4.4.2 2021-10-01
+native_ccache 4.0 2021-11-01
+native_ccache 4.1 2021-11-01
+native_ccache 4.2 2021-11-01
+native_ccache 4.2.1 2021-11-01
+native_ccache 4.3 2021-11-01
+native_ccache 4.4 2021-11-01
+native_ccache 4.4.1 2021-11-01
+native_ccache 4.4.2 2021-11-01
 
 # Clang is currently pinned to LLVM 12
```

Comment:
```suggestion
# Clang is currently pinned to LLVM 13
```

Reply:
Suggestion is incorrect. This file shows postponed upgrades; that is, we haven't upgraded to LLVM 13 yet.